src/extension/logging.c

```diff
@@ -172,23 +184,33 @@ static dd_result _do_dd_log_init() // guarded by mutex
     }
 
     if (_log_strategy != log_use_file || _mlog_fd != -1) {
-        return dd_success;
+        return logging_init_success;
     }
 
+    int at_request = !(!PG(modules_activated) && !PG(during_request_startup));
```

Comment:
The double negative really obfuscates this.

```suggestion
    int at_request = PG(modules_activated) || PG(during_request_startup);
```

Reply:
Basic one 😅

Reply:
Done